src/Microsoft.Extensions.Options.ConfigurationExtensions/ConfigurationChangeTracker.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Extensions.OptionsModel
+{
+    public class ConfigurationChangeTracker<TOptions> : IOptionsChangeTracker<TOptions>
+    {
+        private IConfiguration _config;
+
+        public ConfigurationChangeTracker(IConfiguration config)
```

Comment:
Diego suggested we add Root to the IConfiguration interface so we don't have to pass in the root here.